third_party/nvfuser/csrc/fusion.cpp

```diff
@@ -344,26 +345,150 @@ void Fusion::validateInputs() {
   }
 }
 
-void Fusion::print() {
+void Fusion::serialize(std::ostream& out, SerializationFormat fmt) {
+  FUSER_PERF_SCOPE("Fusion::serialize");
+
+  switch (fmt) {
+    case SerializationFormat::NameOnly:
+      out << "Fusion";
+      break;
+    case SerializationFormat::Default: {
+      FusionGuard fg(this);
+      out << "\n%kernel {\n";
+      IrMathPrinter op_exprs(out);
+      op_exprs.handle(this);
+      out << "\nTransformPrinter : \n";
+      IrTransformPrinter t_exprs(out);
+      t_exprs.handle(this);
+      out << "}\n\n";
+      break;
+    }
+    case SerializationFormat::Debug: {
+      break;
+    }
+    case SerializationFormat::EndOfOption:
+      break;
+  }
+}
+
+void Fusion::printDebug(std::ostream& out) {
+  FUSER_PERF_SCOPE("Fusion::printDebug");
+
+  out << "Fusion DEBUG INFO {";
+  std::vector<Val*> inputs_;
```

Comment:
This vector aliases the fusion's `inputs_` vector so it always shows that the fusion doesn't have any inputs.

Reply:
Yeah, I think we just need to delete this line.